development/gcp/cloud-functions/getgithubcommiter/go.mod

```diff
@@ -0,0 +1,8 @@
+module github.com/kyma-project/test-infra/development/gcp/cloud-functions/getgithubcommiter
+
+go 1.14
```

Comment:
It's 1.14 because this is the golang version I have on my workstation. I will update go.mod files after upgrade on my workstation. Will work on that soon. However, gcp cloud-functions highest supported golang version is 1.13.